prefetch.bs

```diff
@@ -412,7 +413,7 @@ Update all creation sites to supply an empty string, except for any in this docu
 <hr>
 
 <div algorithm="attempt to populate the history entry's document">
-    In <a spec=HTML>attempt to populate the history entry's document</a>, replace the step which invokes <a spec=HTML>create navigation params by fetching</a> with the following:
+    In <a spec=HTML>attempt to populate the history entry's document</a>, add an optional {{Document}} |sourceDocument| argument, and replace the step which invokes <a spec=HTML>create navigation params by fetching</a> with the following:
```

Comment:
Since it is only used for looking up prefetches, I think we should name it something like _sourceDocumentForPrefetches_ or _prefetchSourceDocument_ or similar.

Alternately, maybe we should only plumb the prefetch record through. Then it is more obvious what the purpose of all this is. The downside would be duplicating a couple steps in each call site, hmm.

Reply:
Sounds good > sourceDocumentForPrefetches.

> only plumb the prefetch record ... The downside would be duplicating a couple steps in each call site

+1 > downside -- also https://github.com/WICG/nav-speculation/issues/266 would affect how prefetch records are obtained, so plumbing a Document to https://wicg.github.io/nav-speculation/prefetch.html#find-a-matching-prefetch-record and getting prefetch records there (either from Document / network partition / etc.) is simpler.

Reply:
Done > renaming to _sourceDocumentForPrefetches_.